src/gui/controller/ControllerCreateCase.java

```diff
@@ -0,0 +1,23 @@
+package gui.controller;
+
+import acq.ICase;
+
+/**
+ * A controller used in controllers that is a part of create case
+ */
+public abstract class ControllerCreateCase extends Controller {
```

Comment:
This name is a bit conflicting with `CreateCaseController`, what about `TabController` instead?